### PR TITLE
Feature/create library project

### PR DESCRIPTION
**Pull request by jbern0rd**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -9,34 +9,28 @@ plugins {
...
-}
-
-repositories {
-	mavenLocal()
-	mavenCentral()
-	// iExec
-	maven {
-		url "https://nexus.iex.ec/repository/maven-public/"
+allprojects {
+	group = 'com.iexec.blockchain'
```
Comment (mcornaton) [inline build.gradle:18]: So `group` is in `allprojects` there? :grin:

File: iexec-blockchain-adapter-api-library/src/main/java/com/iexec/blockchain/api/BlockchainAdapterApiClientBuilder.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+import com.iexec.common.utils.FeignBuilder;
+import feign.Logger;
+import feign.auth.BasicAuthRequestInterceptor;
+
+public class BlockchainAdapterApiClientBuilder {
+
+    private BlockchainAdapterApiClientBuilder() {}
+
+    public static BlockchainAdapterApiClient getInstance(Logger.Level logLevel, String url, String username, String password) {
```
Comment (jeremyjams) [inline iexec-blockchain-adapter-api-library/src/main/java/com/iexec/blockchain/api/BlockchainAdapterApiClientBuilder.java:11]: Too long line

Comment (jbern0rd) [reply iexec-blockchain-adapter-api-library/src/main/java/com/iexec/blockchain/api/BlockchainAdapterApiClientBuilder.java]: fixed

**Review by mcornaton — APPROVED** [review]
Looks great, thanks!

File: build.gradle [hunk 3]
```diff
@@ -156,6 +153,6 @@ ext.ociImageName   = 'local/' + ['bash', '-c', 'basename $(git config --get remo
 task buildImage(type: Exec) {
     group 'Build'
     description 'Builds an OCI image from a Dockerfile.'
-    commandLine("sh", "-c", "docker build -f docker/Dockerfile-local --build-arg spring_boot_jar=$bootJarPath -t $ociImageName:$gitShortCommit ."
+    commandLine("sh", "-c", "docker build -f docker/Dockerfile --build-arg spring_boot_jar=$bootJarPath -t $ociImageName:$gitShortCommit ."
```
Comment (zguesmi) [inline build.gradle:156]: No palantir docker extension anymore?

**Review by zguesmi — APPROVED** [review]

File: build.gradle (hunk 3 above)
Comment (jbern0rd) [reply build.gradle:156]: It was removed to converge between all repositories and to be able to quickly copy/paste the `docker build` command if needed.

**Pull request merged**